### PR TITLE
feat: invite link in right drawer

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/8753

Demo: https://www.loom.com/share/49212b2a3b7540d2848f4012ab060c1a

### To test

- [ ] Create a new team and see the right drawer opens with the invite link
- [ ] Create a new org and see the right drawer opens with the invite link

### CODE REVIEW

**Review by acressall — APPROVED** [review]

**Review by Dschoordsch — CHANGES_REQUESTED** [review]
The scrollbar is cut off on initial loading on Safari: Screenshot 2023-09-11 at 12 54 56

File: packages/server/graphql/mutations/addTeam.ts [hunk 1]
```diff
@@ -107,7 +109,17 @@ export default {
...
+      const teamDrawerType: TeamDrawer = 'manageTeam'
+      const [removedSuggestedActionId] = await Promise.all([
+        removeSuggestedAction(viewerId, 'createNewTeam'),
+        r
+          .table('TeamMember')
+          .get(teamMemberId)
+          .update({
+            openDrawer: teamDrawerType
+          })
+          .run()
```
[the remark this answers is not in the document]
Comment (nickoferrall) [reply packages/server/graphql/mutations/addTeam.ts]: I like it! That means less code and more visibility. Updated 👍

**Comment by nickoferrall** [comment]
> The scrollbar is cut off on initial loading on Safari

I am seeing the full scrollbar on initial load in Safari. Are you on linux? 🤔 For me, I think it's a small enough detail that it's OK to merge this PR even if it's not showing up entirely on initial load for everyone 

Screenshot 2023-09-11 at 16 34 54
Screenshot 2023-09-11 at 16 34 33

**Review by Dschoordsch — APPROVED** [review]
I agree it's a small detail and given the small share of users using Safari, that's ok for the first iteration. We should still fix it straight away as the perceived quality is not very good if you see this:

https://github.com/ParabolInc/parabol/assets/7331043/1130ff95-dbc9-4be1-ac94-68780290a3f6

I'm testing on mac

**Comment by acressall** [comment]
The local version is much longer than what will be in production, right? It looks like this version has `invitation....` included, but in prod it should be `prbl.in/[gibberish]` which is more likely to fit

**Comment by nickoferrall** [comment]
> The local version is much longer than what will be in production, right?

Yep, that's right. I can see how it looks with the shorter link. I've created an issue 👍

**Pull request merged**